package.json

```diff
@@ -62,12 +66,14 @@
     "jshint": "^2.10.2",
     "memdown": "^4.1.0",
     "mocha": "^6.2.0",
+    "mysql": "^2.18.1",
     "node-mocks-http": "^1.7.6",
     "nodemon": "^2.0.15",
     "noisegen": "^1.0.0",
     "proxyquire": "^1.7.10",
     "redis-mock": "^0.16.0",
     "rimraf": "^2.6.3",
+    "typescript": "^4.6.2",
```

Comment:
typescript is already installed as dependency, so it could be removed from dev dependencies

Reply:
fixed